samples/Rewrite/ConditionParser/Condition.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Rewrite.ConditionParser
+{
+    public class Condition
+    {
+        public List<ConditionTestStringSegment> TestStringSegments { get; set; }
```

Comment:
Also I think this object could be immutable since it doesn't have to change after you create it